server/strategy.go

```diff
@@ -883,6 +884,11 @@ func rebootOps(c *cke.Cluster, constraints *cke.Constraints, rebootArgs DecideOp
 		return nil, false
 	}
 
+	if len(rebootArgs.RebootCancelled) > 0 {
+		phaseReboot = true
+		ops = append(ops, op.RebootDequeueOp(rebootArgs.RebootCancelled))
```

Comment:
I want to change the display name of the op to `reboot-cancel.`
Is it possible?

Reply:
We can change the display name by duplicating `RebootDequeueOp` and  changing `Name()` method.
The logic of Op is identical. so I created `RebootCancelOp` which has embedded `RebootDequeueOp` and override `Name()` method.